oldScripts/pl_to_spreadsheet.py

```diff
@@ -0,0 +1,102 @@
+# Author: Naveen Nathan
+import time, requests, datetime
+
+import os.path
+import json
+import pandas as pd
```

Comment:
I don't see this used anywhere in the file. For unused variables, it's better to not include them.

Reply:
You're right. I was going to use json and pandas for processing the data and os.path in the Google authentication flow, but none of the libraries are needed for now. .